app/views/upload.scala.html

```diff
@@ -1,6 +1,47 @@
 @()(implicit messages: Messages)
 
 @main(Messages("upload.header")) {
-    <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
-    <p>Placeholder page for upload</p>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <span>
+            <a href="/dashboard" class="govuk-back-link">Back</a>
+            <span class="govuk-caption-xl">Step 3 of 3</span>
+            <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
+        </span>
+        <div class="govuk-form-group">
+            <label class="govuk-label" for="file-upload-1">
+              Select a folder
+            </label>
+            <input class="govuk-file-upload" id="file-upload-1" name="file-upload-1" type="file">
```

Comment:
Is this limited to just directories, rather than individual files? I forget exactly how we did this in the prototype - it might be the `webkitdirectory` attribute here: https://github.com/nationalarchives/tdr-prototype-mvc/blob/29332ab8b1d0fdbbcc3ca186ec88cfdc0545e2d2/app/views/upload.scala.html#L39

See https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory